openhtmltopdf-core/src/main/java/com/openhtmltopdf/render/BlockBox.java

```diff
@@ -428,6 +430,73 @@ private MarkerData.TextMarker makeTextMarker(LayoutContext c, IdentValue listSty
 
         MarkerData.TextMarker result = new MarkerData.TextMarker();
 
+        result.setAlignment(IdentValue.END);
+        result.setLayoutWidth(w);
+        result.setText(text);
+
+        return result;
+    }
+
+    private MarkerData.TextMarker makeTextMarker(LayoutContext c, CascadedStyle markerStyle) {
+        IdentValue listDirection = getParent().getStyle().getDirection();
+        IdentValue alignment = getStyle().deriveStyle(markerStyle).getTextAlign();
```

Comment:
Should `alignment` be initialized here before the following if/else chain? It looks like this initial value will be overridden no matter what.

Reply:
Yeah, I think this is probably a hangover from before I backfilled RTL and can be removed.